SMTPServer/SMTPServer.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <iostream>
+
+#include <winsock2.h>
+#include <ws2tcpip.h>
+
+#include <memory>
+#include <thread>
+#include <string>
+
+#include "ThreadPool.h"
+#include "MailSession.h"
+
+#pragma comment(lib, "Ws2_32.lib")
```

Comment:
You have already added Ws2_32.lib in MailSession.h (the line 10).
Linked will give warning about this issue.